### ISSUE TITLE
V 1.43 does not launch on macOS Mojave

### ISSUE THREAD

**Issue by nikohansen** [microsoft/vscode#94391]
- VSCode Version: 1.43 updated as of 2020/04/03 (probably 1.43.2)
- OS Version: macOS Mojave 10.14.6

Steps to Reproduce:

1. download and unzip latest version 
2. click on vscode 
3. nothing happens, also no blocking message under the macOS privacy settings

It first happened when I clicked on "restart to update" that vscode stop to launch altogether.

Fix: back to version 1.42 works.

My suspicion is that it has to do with the codesigning issue.

Does this issue occur when all extensions are disabled?: I am running version 1.42 and don't know how to invoke the latest version via "code".

**Comment by joaomoreno** [comment]
Did you check the privacy section in Settings?

What happens if you run it from the command line?

```
./path/to/Visual Studio Code.app/Contents/MacOS/Electron
```

**Comment by nikohansen** [comment]
> Did you check the privacy section in Settings?

Not sure what you mean above and beyond of what I wrote above

> 3. nothing happens, also no blocking message under the macOS privacy settings

> What happens if you run it from the command line?

Thanks, that works indeed. WIth a new download I also could open it once by clicking (with a warning message for opening a new app), once I got a warning that another vscode instance is running, and now it's back to clicking the app does nothing. Command line works. Having written that, now the version downloaded previously residing in the Application folder _does_ open on double click. OK, strange, that's good enough for me, if it's for you ;-)
